_sub/compute/eks-nodegroup-unmanaged/main.tf

```diff
@@ -98,3 +53,13 @@ resource "aws_autoscaling_group" "eks" {
 
 }
 
+
+resource "aws_autoscaling_schedule" "eks" {
+  count                  = var.is_sandbox && length(var.instance_types) > 0 ? 1 : 0
+  autoscaling_group_name = aws_autoscaling_group.eks[0].name
+  scheduled_action_name  = "Scale to zero"
+  desired_capacity       = 0
```

Comment:
Hardcoded to zero?

Reply:
Because `scheduled_action_name  = "Scale to zero"` ;)